### PR TITLE
Handles the issue of finding the starting days.

### PR DESCRIPTION
**Pull request by dpspcghst**
The counting_days file contains a description of the problem from projecteuler.net. It also contains the get_days function, which iterates over the years between 1901 and 2000, and returns a string of the day of the week if the generated weekday() integer matches an item on the "days" list.

### CODE REVIEW

File: counting_days.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Euler Problem #19: https://projecteuler.net/problem=19
+
+You are given the following information, but you may prefer to do some
+research for yourself.
+
+    * 1 Jan 1900 was a Monday.
+    * Thirty days has September, April, June and November. All the rest
+      have thirty-one, Saving February alone, Which has twenty-eight, rain
+      or shine. And on leap years, twenty-nine.
+    * A leap year occurs on any year evenly divisible by 4, but not on a
```
Comment (jabrielcjones) [inline counting_days.py:11]: I didn't know there were centuries that did not have leap years. Good to know

File: counting_days.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    months = list(range(1, 12))
+
+    days = [
+        "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"
+    ]
+
+    for year in years:
+        for month in months:
+
+            if date(year, month, 1).weekday() == 0:
```
Comment (jabrielcjones) [inline counting_days.py:39]: You probably can reduce the entire if-else block to a single line: `days[date(year, month, 1).weekday()]`

Comment (jabrielcjones) [reply counting_days.py:39]: You would have an `if` and an `else` and that's it

Comment (deepspaceghost) [reply counting_days.py:39]: I put your suggestion to good use. It works like a charm. Thanks for the tip!

**Pull request closed without merging**